DiceKitTests/Die_Tests.swift

```diff
@@ -404,4 +404,38 @@ extension Die_Tests {
         expect(result.value) == stubbedRoll
     }
     
+    func test_probabilityMass_shouldReturnCorrect() {
+        property("probability mass") <- forAll {
+            (sides: Int) in
+            
+            let inverseSides = -sides
+            
+            let die = Die(sides: sides)
+            let inverseDie = Die(sides: inverseSides)
+            
+            let probMass = die.probabilityMass
+            let inverseProbMass = inverseDie.probabilityMass
+            
+            guard sides != 0 else {
```

Comment:
Kinda strange to have the `guard` statement way down here, no? There is a lot more going on in this test so I can kinda understand the spacing above.